src/proto/active_packetizer.rs

```diff
@@ -415,9 +410,13 @@ where
                         } = r
                         {
                             assert!(timeout >= 0);
-                            trace!(logger, "negotiated session timeout: {}ms", timeout);
 
                             *this.timeout = time::Duration::from_millis(2 * timeout as u64 / 3);
+                            trace!(
```

Comment:
nitpick: The new trace doesn't indicate which unit (ms) the timeout uses. Maybe add that as a field or as part of the message?

Reply:
I moved the trace around, so that we emit the std::time::Duration instead of an u64.
It now shows up as `negotiated session timeout timeout=2.666s`

Reply:
Ah neat! Then consider this as resolved.